src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table
+public class Product {
+
+    private long id;
+    private Order order;
```

Comment:
Zamówienie może się składać z wielu produktów - to się zgadza, ale czy jeden produkt nie może występować w wielu zamówieniach?